README.md

```diff
@@ -1 +1,36 @@
-# Capstone1
\ No newline at end of file
+# Youth Week
+
+> a website about the International youth Week event. for the microverse capstone1.
+
+## Built With
+
+- HTML
+- SCSS
+- JavaScript
+
+## Live Project
+
+This site was built using [GitHub Pages](https://ismailakinkunmi.github.io/Capstone1/)
+
+## Video Presentation
+
+[Loom Link](https://www.loom.com/share/74a5724e4b0e46b299357fd4fd807371/)
```

Comment:
- [ ] Thank you for adding a video demo. However, you are required to be visible in the video as you explain what you did. 
- [ ] remember to highlight a small piece of code you are proud of.
- [ ] [OPTIONAL] please try to speak a bit loader and more clearly as it is somewhat difficult to hear you explain.